### DISCUSSION TITLE
Biomass Mantainance

### DISCUSSION THREAD

**Discussion by fgualdr** [SysBioChalmers/Human-GEM#238] [Q&A] [answered]
Hi,

I am trying to use HumanGEM to perform cell type tailoring and FBA.
the cell type I am trying to model is a resting non-growing cell.
Could I ask if inbuilt in human there is a biomass mantainance that I can use?
Thanks in advance
Francesco

**Comment by JonathanRob** [comment, marked as answer]
Hi @fgualdr, yes, there are a few options one could use for maintenance:
- `biomass_maintenance_Recon3D` and `biomass_maintenance_noTrTr_Recon3D` originate from the Recon3D model, and could be used (the latter is without transcription or translation). Note that these reactions are currently inactivated in Human-GEM (`ub = 0`), so don't forget to open their bounds before using.
- `HMR_3964` is just basic ATP hydrolysis, if you're interested in a more minimal maintenance cost.

**Comment by haowang-bioinfo** [comment]
@fgualdr Yes, Human-GEM includes several different `biomass` objective functions that were described in #63.

**Comment by JonathanRob** [comment]
I will convert this Issue to a "Discussion", since I think that is a better fit.
